invenio/modules/deposit/templates/deposit/widget_plupload_base.html

```diff
@@ -50,7 +50,7 @@
     <tbody id="sortable">
     </tbody>
   </table>
-  <p class="text-muted" align="center"><small>Drag and drop files to upload</small></p>
+  <p id="field-plupload_file" class="text-muted" align="center"><small>Drag and drop files to upload</small></p>
```

Comment:
I've only added the id that is requested by some JavaScript code which is not mine. If the code isn't there, it seems to fail (JS error).